src/resources/js/routes.js

```diff
@@ -2,14 +2,19 @@ import Vue from 'vue';
 import VueRouter from 'vue-router';
 
 import Hello from './components/Hello';
+import Authentication from './components/Authentication';
+import AccountEdition from './components/AccountEdition';
 
 Vue.use(VueRouter);
 
 export const router = new VueRouter({
   mode: 'history',
-    routes: [{
-        path: '/hello',
-        name: 'Hello',
-        component: Hello
-    }],
+    routes: [
+    {
+        path: '/user/edit',
```

Comment:
the path should be in french, as our app is in french, and that's something the user can see.

Reply:
no